Gemfile.lock

```diff
@@ -134,13 +140,13 @@ GEM
       govuk_frontend_toolkit (>= 6.0.2)
       rails (>= 4.1.0)
       sass (>= 3.2.0)
-    govuk_frontend_toolkit (7.6.0)
+    govuk_frontend_toolkit (8.1.0)
```

Comment:
Can we exclude the upgrade for `govuk_frontend_toolkit`? I'd first like to check the UA tracking doesn't break anything for Cliff.